src/components/executive-education-2u/UserEnrollmentForm.jsx

```diff
@@ -42,6 +45,14 @@ function UserEnrollmentForm({
     if (!values.studentTermsAndConditions) {
       errors.studentTermsAndConditions = formValidationMessages.studentTermsAndConditionsRequired;
     }
+
+    if (errors) {
+      sendEnterpriseTrackEvent(
+        enterpriseId,
+        'edx.ui.enterprise.learner_portal.executive_education.checkout_form.validation.failed',
+        { errors },
+      );
+    }
```

Comment:
[curious] How often does this function get run? The form is set to validate only upon click of the submit button, but then, it switches to do "inline" validation after the first submission. I think my concern here _might_ be that we're over calling `sendEnterpriseTrackEvent` on every keystroke thereafter initial form submission, if that makes sense.

Wondering if this event needs to be debounced/throttled in any way?

Reply:
Synced and decided to track only the initial validation errors.